sanitize.schema.js

```diff
@@ -1,33 +1,42 @@
-const { defaultSchema: sanitize } = require('hast-util-sanitize/lib/schema');
+const { defaultSchema } = require('hast-util-sanitize/lib/schema');
 
-// Sanitization Schema Defaults
-sanitize.clobberPrefix = '';
+const createSchema = ({ safeMode } = {}) => {
+  const schema = JSON.parse(JSON.stringify(defaultSchema));
```

Comment:
Stringifying and parsing so you can clone `defaultSchema`?

Reply:
Yea, it's imported as a single object. The reason some of our tests were passing was because a previous one set the correct expected configuration first.